buildSrc/build.gradle.kts

```diff
@@ -12,5 +12,5 @@ dependencies {
 	implementation("de.marcphilipp.gradle:nexus-publish-plugin:0.4.0")
 	implementation("biz.aQute.bnd:biz.aQute.bnd.gradle:5.2.0")
 	implementation("com.github.jengelman.gradle.plugins:shadow:6.1.0")
-	implementation("org.gradle:test-retry-gradle-plugin:1.1.8")
+	implementation("org.gradle:test-retry-gradle-plugin:1.1.9")
```

Comment:
I have updated this dependency because of tests were failing on old ASM version that is not able to read bytecode version 59